src/shared/parse-helpers.h

```diff
@@ -23,3 +39,10 @@ int parse_socket_bind_item(
         int *ip_protocol,
         uint16_t *nr_ports,
         uint16_t *port_min);
+
+int parse_open_file(
+        const char *v,
+        OpenFile* of);
+
+const char *open_file_to_string(const OpenFile *open_file);
```

Comment:
I would just use `of` as the default variable name for `OpenFile` local variables and input arguments everywhere to keep things concise.